app/services/check_answers/sections/housing_and_property.rb

```diff
@@ -0,0 +1,97 @@
+                                     FieldPresenter.new(table_label: :property, attribute: :property_owned, type: :select, model: @check, partner_dependant_wording: true),
+                                   ])
+        housing_costs = unless @check.owns_property_outright?
+                          Table.new(screen: :housing_costs, skip_change_link: false, index: nil, disputed?: nil,
+                                    fields: housing_costs_fields(:housing_costs))
+                        end
+        mort_table = if @check.property_owned_with_mortgage?
+                       Table.new(screen: :mortgage_or_loan_payment, skip_change_link: false, index: nil, disputed?: nil,
+                                 fields: [
+                                   MoneyWithFrequencyPresenter.new(table_label: :mortgage_or_loan_payment, attribute: :housing_loan_payments, model: @check,
+                                                                   frequency_attribute: :housing_payments_loan_frequency),
+
+                                 ])
+                     end
+        prop_entry_table = if @check.owns_property?
+                             Table.new(screen: :property_entry, skip_change_link: false, index: nil, disputed?: @check.house_in_dispute,
+                                       fields: [
+                                         MoneyPresenter.new(table_label: :property_entry, attribute: :house_value, model: @check),
+                                         if @check.property_owned_with_mortgage?
+                                           MoneyPresenter.new(table_label: :property_entry, attribute: :mortgage, model: @check)
+                                         end,
+                                         FieldPresenter.new(table_label: :property_entry, attribute: :percentage_owned, type: :percentage, model: @check),
+                                       ].compact)
+                           end
+
+        [property_table, housing_costs, mort_table, prop_entry_table].compact
+      end
+
+      def housing_costs_fields(label)
+        [
+          MoneyWithFrequencyPresenter.new(table_label: label, attribute: :housing_payments, model: @check,
+                                          frequency_attribute: :housing_payments_frequency),
+          FieldPresenter.new(table_label: label, attribute: :housing_benefit_relevant, type: :boolean, model: @check),
+          if @check.housing_benefit_relevant?
+            MoneyWithFrequencyPresenter.new(table_label: label, attribute: :housing_benefit_value, model: @check,
+                                            frequency_attribute: :housing_benefit_frequency)
+          end,
+        ].compact
+      end
+
+      def additional_property
+        additional_property_tables :additional_property, @check.additional_properties,
+                                   :additional_property_details, :additional_property_owned
+      end
+
+      def partner_additional_property
```

Comment:
Does this need to be on the condition of a partner existing?  I get this extra table (marked in red) when I remove a partner. 
![Screenshot 2024-08-12 at 13 40 58](https://github.com/user-attachments/assets/8fefd52d-c12e-4db5-b430-a5202067e6f3)